src/components/Proposals/ProposalPage/OPProposalPage/ProposalVotesBar/proposalVotesBar.module.scss

```diff
@@ -1,6 +1,7 @@
 @import "@/styles/variables.scss";
 
 .vote_bar_ticks {
+  cursor: pointer;
```

Comment:
`cursor: help` is perhaps more appropriate here

Reply:
Good idea, done! LGTM :)